.github/workflows/update_year_number.yml

```diff
@@ -0,0 +1,33 @@
+name: Update year number
+
+# Trigger this action manually
+on:
+  workflow_dispatch:
+    inputs:
+      year_number:
+        required: true
+
+jobs:
+  updating_year_number:
+    runs-on: macos-latest
+    steps:
+# Getting files of OpenMS
+      - uses: actions/checkout@v3
+
+# Find all files and replace the year number
+      - name: Update files
+        run: |
+          # setting a variable for year number
+          year_number="${{ github.event.inputs.year_number }}"
+          # Find and replace in all files
+          find . -type f ! -path "./.git/*" -exec grep -q "Freie Universitaet Berlin 2002-20" {} \; -exec sed -i '' "s#and Freie Universitaet Berlin 2002-20.*.#and Freie Universitaet Berlin 2002-$year_number.#g" {} \;
```

Comment:
I know this probably works but it would be safer and easier to extend later if it were something like 2002-20[0-9]{2}

Reply:
Fixed!